systemtests/src/main/java/com/github/streamshub/systemtests/setup/StrimziOperatorSetup.java

```diff
@@ -177,12 +168,16 @@ private Deployment getBundleDeployment() {
                     .endSpec()
                 .endTemplate()
             .endSpec()
-            .build();
+            .build())
+            .toList()
+            .get(0);
```

Comment:
```suggestion
            .findFirst()
            .orElseThrow();
```

Reply:
Thank you, don't know how i missed that.